tensorboard/plugins/audio/summary.py

```diff
@@ -201,7 +201,7 @@ def pb(name,
 
   encoded_audio = [encoder(a) for a in limited_audio]
   content = np.array([encoded_audio, limited_labels]).transpose()
-  tensor = tf.compat.v1.make_tensor_proto(content, dtype=tf.string)
+  tensor = tf.make_tensor_proto(content, dtype=tf.string)
```

Comment:
It doesn’t look to me like `tf.make_tensor_proto` exists on the latest
nightly 2.0 preview:

    $ pip install tf-nightly-2.0-preview==1.13.0.dev20181220
    $ python
    >>> import tensorflow as tf
    >>> tf.version.VERSION
    '1.13.0-dev20181220'
    >>> tf.make_tensor_proto
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    AttributeError: 'module' object has no attribute 'make_tensor_proto'

Is this code 2.0-compatible?

Reply:
As we discussed offline, many `plugins/*/summary.py` do `import tensorflow.compat.v1 as tf` and TF migration script does not take that into account. Above code before change was effectively doing `tf.compat.v1.compat.v1.make_tensor_proto`.